compiler/quilt/tools/store.py

```diff
@@ -40,7 +39,7 @@ class and its subclasses abstract file formats, file naming and
     TMP_OBJ_DIR = 'tmp'
     PKG_DIR = 'pkgs'
     CACHE_DIR = 'cache'
-    VERSION = '1.2'
+    VERSION = '1.3'
```

Comment:
Does it mean, users will have to blow away their `quilt_packages` again?

Reply:
Yes.